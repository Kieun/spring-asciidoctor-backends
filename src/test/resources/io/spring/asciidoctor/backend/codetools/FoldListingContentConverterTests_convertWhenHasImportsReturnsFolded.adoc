= Test Document

[source,java]
----
import java.net.InetAddress;
import java.util.List;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.ConstructorBinding;
import org.springframework.boot.context.properties.bind.DefaultValue;

public class Example {

	private final String first;

	private final String second;

	public Example(String first, String second) {
		this.first = first;
		this.second = second;
	}

}
----
