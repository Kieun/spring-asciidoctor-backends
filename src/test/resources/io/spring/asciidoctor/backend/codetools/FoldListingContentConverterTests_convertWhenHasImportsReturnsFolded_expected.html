<span class="fold-block hide-when-folded">import java.net.InetAddress;
import java.util.List;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.ConstructorBinding;
import org.springframework.boot.context.properties.bind.DefaultValue;

</span><span class="fold-block">public class Example {

	private final String first;

	private final String second;

	public Example(String first, String second) {
		this.first = first;
		this.second = second;
	}

}
</span>